essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md

```diff
@@ -44,44 +46,64 @@ yay -S curl base-devel sqlite openssl clang cmake ninja pkg-config gtk3 unzip
 {% endtab %}
 {% endtabs %}
 
+![img.png](../../../../.gitbook/assets/build\_step\_two.png)
+
 * Install flutter according to [https://docs.flutter.dev/get-started/install/linux](https://docs.flutter.dev/get-started/install/linux). Make sure to install flutter in a directory that is appropriate for you.
 
+* Or you can use the code below to install `FLUTTER` manually on your linux system.
+
 ```bash
 git clone https://github.com/flutter/flutter.git
 cd flutter
 echo "export PATH=\$PATH:"`pwd`"/bin" >> ~/.profile
 export PATH="$PATH:`pwd`/bin"
+cd ..
 ```
+![img.png](../../../../.gitbook/assets/flutter\_build\_step.png)
 
 * Run the setup script from the base directory
 ```bash
+cd appflowy
 ./frontend/scripts/install_dev_env/install_linux.sh
 ```
+![img.png](../../../../.gitbook/assets/build\_step\_two\_script.png)
+
+![img.png](../../../../.gitbook/assets/build\_step\_two\_warn\_success.png)
+
+`If you get the warning as shown above run following command`
```

Comment:
If you get the warning as shown above, run the following command:

Reply:
Should I remove the "`"? or just the punctuation mark missing?